exe/dash_timeline_validator

```diff
@@ -1,10 +1,34 @@
 #!/usr/bin/env ruby
 
+require "optparse"
 require "dash_timeline_validator"
 
+options = {}
+OptionParser.new do |opts|
+    opts.banner = "DASH Timeline Validator"  
+    opts.on("--acceptable_drift=ACCEPTABLE_DRIFT", "Minimum duration drift acceptable between the sequential segments", Integer) do |v|
+      options[:acceptable_drift] = v
+    end
+    opts.on("--presentation_delay=PRESENTATION_DELAY", "Delay in seconds of the live edge", Integer) do |v|
+      options[:presentation_delay] = v
+    end
+    opts.on("--buffered_segments=BUFFERED_SEGMENTS", "Number of segments buffered by the player to generate the live edge", Integer) do |v|
+      options[:buffered_segments] = v
+    end
+    opts.on("--verify_segments_duration=VERIFY_SEGMENTS_DURATION", "Check the duration of every segment when setted to true") do |v|
+      options[:verify_segments_duration] = v
+    end
+    opts.on("--analyzer_folder=ANALYZER_FOLDER", "Folder used to download the files") do |v|
+      options[:analyzer_folder] = v
+    end
+    opts.on("--analyzer_manifest_path=ANALYZER_MANIFEST_PATH", "Manifest path") do |v|
+      options[:analyzer_manifest_path] = v
+    end
+end.parse!
+
 if ARGV.length == 0
   p "You must pass the manifest!"
```

Comment:
When nothing is passed, the banner should be displayed instead of this message.

Reply:
Done